### PR TITLE
fix(gradle): Upgraded to gradle 3.2

### PR DESCRIPTION
**Pull request by yyekhlef**
updated build.gradle: upgraded spring-boot-gradle-plugin to 1.4.2.RELEASE to make it compatible with gradle 3.2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Mini webservice with spring-boot … [![Build Status](https://travis-ci.org/vdemeester/miniws.svg)](https://travis-ci.org/vdemeester/miniws)
+# Mini webservice with spring-boot … [![Build Status](https://travis-ci.org/yyekhlef/miniws.svg?branch=master)](https://travis-ci.org/yyekhlef/miniws.svg?branch=master)
```
Comment (vdemeester) [inline README.md:1]: Are these changes required ? 😝

**Review by vdemeester — APPROVED** [review]
LGTM 🐸 
Just a question

File: README.md (hunk 1 above)
Comment (yyekhlef) [reply README.md:1]: Nope, I just wanted to 🔫 you. :-)

Comment (vdemeester) [reply README.md:1]: 😂

**Comment by yyekhlef** [comment]
LGTM
